### PR TITLE
Add I18n settings to toolbar

### PR DESCRIPTION
**Pull request by awead**
Add I18n terms to the toolbar.

@projecthydra/sufia-code-reviewers

### CODE REVIEW

File: app/views/_toolbar.html.erb [hunk 1]
```diff
@@ -3,65 +3,65 @@
...
-          <span class="fa fa-gear"></span> Admin <span class="caret"></span>
+          <span class="fa fa-gear"></span> <%= t("sufia.toolbar.admin.menu") %> <span class="caret"></span>
         <% end %>
         <ul class="dropdown-menu">
           <% if respond_to? :admin_stats_path %>
-            <li><%= link_to 'Repository Statistics', sufia.admin_stats_path %></li>
+            <li><%= link_to t("sufia.toolbar.admin.statistics"), sufia.admin_stats_path %></li>
           <% end %>
 
           <% if respond_to? :admin_queues_path %>
```
Comment (jcoyne) [inline app/views/_toolbar.html.erb]: Can we just eliminate this one? it doesn't respond to this anymore.

Comment (awead) [inline app/views/_toolbar.html.erb]: Alternatively, could we provide a blank partial that uses could override with links to whatever queuing service they're using?

Comment (jcoyne) [inline app/views/_toolbar.html.erb]: I'm not a fan of that. I wouldn't mind breaking the admin dropdown partial out. Then you'd only have a few lines to duplicate if you choose to add additional items.

Comment (awead) [inline app/views/_toolbar.html.erb]: That's sounds even better.

File: config/locales/sufia.en.yml [hunk 2]
```diff
@@ -73,6 +73,26 @@ en:
         details:            'Details'
         tab_label:          'Recently Uploaded'
         no_public:          'No public work has been contributed.'
+    toolbar:
+      admin:
+        menu: "Admin"
+        statistics: "Repository Statistics"
+        queues: "Message Queues"
```
Comment (jcoyne) [inline config/locales/sufia.en.yml]: Remove this.

File: app/views/admin/_menu.html.erb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<li class="dropdown">
+  <%= link_to sufia.admin_stats_path, role: 'button', data: { toggle: 'dropdown' }, aria: { haspopup: true, expanded: false } do %>
+    <span class="fa fa-gear"></span> <%= t("sufia.toolbar.admin.menu") %> <span class="caret"></span>
+  <% end %>
+  <ul class="dropdown-menu">
+    <% if respond_to? :admin_stats_path %>
```
Comment (jcoyne) [inline app/views/admin/_menu.html.erb]: Can we remove this conditional? It should always be true.

Comment (mjgiarlo) [inline app/views/admin/_menu.html.erb]: Yes, this is now true. Related: we should remove this conditional as well: https://github.com/projecthydra/sufia/blob/master/config/routes.rb#L153

I'd be fine with seeing that snuck into this PR.

Comment (awead) [inline app/views/admin/_menu.html.erb]: See #2183

**Pull request merged**